OracleDatabase/RAC/OracleDNSServer/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# 
+# Since: November, 2018
+# Author: paramdeep.saini@oracle.com
+# Description: Build script for building Oracle Database RAC Docker images.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+# Copyright (c) 2014-2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This copyright date format is wrong (it should be `2014, 2021`) and we no longer use "All rights reserved." so that should be removed from all the copyright statements.